crates/bevy_ecs/src/schedule/executor/mod.rs

```diff
@@ -105,11 +107,14 @@ pub struct SystemSchedule {
     ///
     /// If a set doesn't run because of its conditions, this is used to skip all systems in it.
     pub(super) systems_in_sets_with_conditions: Vec<FixedBitSet>,
+    /// Sparse mapping of system set node id to systems in the set, used for running
+    /// subgraphs. This is filled lazily when a subgraph is run for the first time.
+    pub(super) systems_in_sets: HashMap<SystemSetKey, FixedBitSet>,
 }
 
 impl SystemSchedule {
     /// Creates an empty [`SystemSchedule`].
-    pub const fn new() -> Self {
```

Comment:
This is a breaking change. Do we know why this was made const in the first place?